### PR TITLE
Markdown of financial annual report

### PR DESCRIPTION
**Pull request by ElenaFdR**
Now with tables in Markdown, too.

-----
[View rendered organization/annual-reports/financial-report-2021.md](https://github.com/publiccodenet/about/blob/financial-annual-report-2021/organization/annual-reports/financial-report-2021.md)

### CODE REVIEW

File: organization/annual-reports/financial-report-2021.md [hunk 1]
```diff
@@ -0,0 +1,559 @@
+---
+type: resource
+---
+
+
+
```
Comment (ericherman) [inline organization/annual-reports/financial-report-2021.md:6]: lines 4, 5, and 6 are all blank. I expect those should be collapased to a single blank line

Comment (ElenaFdR) [reply organization/annual-reports/financial-report-2021.md]: My delinter didn't pick that up!

File: organization/annual-reports/financial-report-2021.md [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+Public code stewarded by the Foundation is openly available for reuse by everyone, but only members (wholly publicly owned organizations) can determine the organization’s governance and the stewardship priorities.
+
+The association seeks to achieve its goal by carrying out the following activities (article 3.2 of the association bylaws):
+
+* 1. the provision of services that can increase sustainability, quality and re-use of public code
+* 2. facilitating and encouraging the sharing of existing public code between public members and other public organizations
+* 3. carrying out research for the (further) development of public code
+* 4. drawing attention to problems ascertained and giving advice with regard to the public code they use
+* 5. everything related to or conducive to the above, all in the broadest sense
```
Comment (ericherman) [inline organization/annual-reports/financial-report-2021.md:37]: I don't think we want these to be both bullets and numbers

File: organization/annual-reports/financial-report-2021.md [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+###### Beneficiary of the liquidation balance
+
+If the Foundation for Public Code’s Board of Directors decides to dissolve the association, the association will continue to exist for as long as is necessary to liquidate the association’s assets. Article 31.4 of the Foundation for Public Code’s bylaws state that any positive balance at the time of liquidation will be given to either:
+
+* a public benefit organization (ANBI) with a similar goal to the Foundation for Public Code
+* a foreign organization dedicated exclusively or nearly completely to the public benefit with a similar goal to the Foundation for Public Code
+
+##### 1.1.3 Mission and vision
+
+The Foundation for Public Code’s mission is “to support collective development and maintenance of public code in order to reduce risk, increase quality, reduce cost and increase sustainability”.” (established in article 3.1 of the bylaws).
```
Comment (ericherman) [inline organization/annual-reports/financial-report-2021.md:52]: extra double-quote at sentence end

File: organization/annual-reports/financial-report-2021.md (hunk 2 above)
Comment (ElenaFdR) [reply organization/annual-reports/financial-report-2021.md:37]: This is a [straight copy from our articles of association](https://about.publiccode.net/organization/articles-of-association.html#article-3-purpose-and-methodology), including the formatting.

Comment (ElenaFdR) [reply organization/annual-reports/financial-report-2021.md:37]: Sorry, I'm wrong. I'll play around with it.

File: organization/annual-reports/financial-report-2021.md [hunk 4]
```diff
@@ -0,0 +1,557 @@
...
+| **Total** | **636,706** | **652,249** |
+| **Operating result** | **-135,100** | **14,302** |
+| Financial gains and losses | -4,771 | -39,787 |
+| **Total** | **-139,871** | **-25,485** |
+
+#### 5.3 Notes to the financial statements
+
+##### General
+
+These are the second financial statements of The Foundation for Public Code Vereniging. The book year ran from 1 January 2020 - 31 December 2021.
```
Comment (Ainali) [inline organization/annual-reports/financial-report-2021.md:327]:
```suggestion
These are the second financial statements of The Foundation for Public Code Vereniging. The book year ran from 1 January 2020 - 31 December 2021.
```
```suggestion
These are the second financial statements of The Foundation for Public Code Vereniging. The book year ran from 1 January 2021 - 31 December 2021.
```

File: organization/annual-reports/financial-report-2021.md [hunk 5]
```diff
@@ -0,0 +1,557 @@
...
+
+The Foundation for Public Code will deliver its mission by:
+
+* enabling an ecosystem of Public Code software products, developed by and for public administrations to solve real problems for citizens
+* building a collaborative community of developers and public organizations, which continuously improves and maintains public code products
+* providing a certification system for software as [meeting the standard](https://standard.publiccode.net/) set by the Foundation for Public Code
+* building a trusted brand and an awareness of the value of public code.
+
+The Foundation for Public Code is completely unique. There is no other organization that provides impartial codebase stewardship services designed to support open collaboration of public organizations for the benefit of citizens and taxpayers. This distinguishes it from organizations providing open source codebase stewardship for private sector projects, from vendors selling open source software development to public organizations, and from activist groups lobbying governments to spend taxpayer money only on open source software.
```
Comment (ericherman) [inline organization/annual-reports/financial-report-2021.md:61]: Should "impartial codebase stewardship _services_" be replaced with "impartial codebase stewardship _activities_"?

**Review by Ainali — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Pull request merged**